cli.js

```diff
@@ -69,10 +70,11 @@ program
 	.option('--ip <ip_addr>', 'ip address of device')
 	.option('--id <id>', 'id of device')
 	.option('--key [key]', 'key of device')
+	.option('--cid [cid_zigbee_device]', 'if specified, use device id of zigbee gateway and cid of subdevice to set its property')
 	.option('--set <set>', 'value to set')
 	.option('--raw-value', 'pass the raw set value without attempting to parse it from a string')
 	.option('--dps [dps]', 'DPS index to set', 1)
-	.option('--protocol-version [version]', 'tuya protocol version', parseFloat, 3.1)
```

Comment:
This is breaking, probably should leave it as 3.1 for now.

Reply:
Done!